generator/views/funnel_analysis_view.py

```diff
@@ -160,22 +160,30 @@ def _event_types_lookml(self) -> List[Dict[str, Any]]:
                 {
                     "name": "event_types",
                     "derived_table": {
-                        "sql": (
-                            "SELECT "
-                            "mozfun.event_analysis.aggregate_match_strings( "
-                            "ARRAY_AGG(CONCAT(COALESCE("
-                            "mozfun.event_analysis.escape_metachars(property_value.value), ''),"
-                            "mozfun.event_analysis.event_index_to_match_string(et.index)))) AS match_string "
-                            "FROM "
-                            f"{self.tables[0]['event_types']} as et "
-                            "LEFT JOIN UNNEST(COALESCE(event_properties, [])) AS properties "
-                            "LEFT JOIN UNNEST(properties.value) AS property_value "
-                            "WHERE "
-                            "{% condition category %} category {% endcondition %} "
-                            "AND {% condition event %} event {% endcondition %} "
-                            "AND {% condition property_name %} properties.key {% endcondition %} "
-                            "AND {% condition property_value %} property_value.key {% endcondition %}"
-                        )
+                        "sql": dedent(
+                            f"""
+                            SELECT
+                                mozfun.event_analysis.aggregate_match_strings(
+                                    ARRAY_AGG(
+                                        CONCAT(
+                                            COALESCE(
+                                                mozfun.event_analysis.escape_metachars(property_value.value), '')
+                                            ),
+                                            mozfun.event_analysis.event_index_to_match_string(et.index)
+                                        )
+                                    )
+                                ) AS match_string
+                            FROM
+                                {self.tables[0]['event_types']} as et
+                                LEFT JOIN UNNEST(COALESCE(event_properties, [])) AS properties
+                                LEFT JOIN UNNEST(properties.value) AS property_value
+                            WHERE
+                                {{% condition category %}} category {{% endcondition %}}
+                                AND {{% condition event %}} event {{% endcondition %}}
+                                AND {{% condition property_name %}} properties.key {{% endcondition %}}
+                                AND {{% condition property_value %}} property_value.key {{% endcondition %}}
```

Comment:
Kind of a gray area with this being a Python file where 4-space indents are standard, but I'd lean toward using 2-space indents within the SQL string to match our `bqetl format` standards.
```suggestion
                            SELECT
                              mozfun.event_analysis.aggregate_match_strings(
                                ARRAY_AGG(
                                  CONCAT(
                                    COALESCE(
                                      mozfun.event_analysis.escape_metachars(property_value.value),
                                      ''
                                    ),
                                    mozfun.event_analysis.event_index_to_match_string(et.index)
                                  )
                                )
                              ) AS match_string
                            FROM
                              {self.tables[0]['event_types']} AS et
                            LEFT JOIN
                              UNNEST(COALESCE(event_properties, [])) AS properties
                            LEFT JOIN
                              UNNEST(properties.value) AS property_value
                            WHERE
                              {{% condition category %}} category {{% endcondition %}}
                              AND {{% condition event %}} event {{% endcondition %}}
                              AND {{% condition property_name %}} properties.key {{% endcondition %}}
                              AND {{% condition property_value %}} property_value.key {{% endcondition %}}
```